includes/commands/chungus.js

```diff
@@ -730,6 +731,25 @@ function checkHeldTime(msg, args) {
 	});
 }
 
+function currentStreak(userId, callback){
+	db.get(
+		//There has to be a better way to only get the most recent row where "BecameChungus=1" but I don't know the db well enough to craft it.
```

Comment:
You can indeed make it a bit simpler. Since you are getting data exclusively for a specific userId, theres no need to ensure the server is accurate, and thus dont need to join the user table.

```
SELECT DateCreated
FROM ChungusPoints
WHERE UserId = ${userId} AND BecameChungus = 1
ORDER BY DateCreated DESC
LIMIT 1
```